react/components/InfoCard/index.js

```diff
@@ -156,55 +156,59 @@ const InfoCard = ({
     [intl, subhead]
   )
 
-  return (
-    <LinkWrapper
-      imageActionUrl={formatIOMessage({ id: imageActionUrl, intl })}
-      extraCondition={!isFullModeStyle}
-      linkProps={{ className: 'no-underline' }}
-    >
-      <div
-        className={containerClasses}
-        style={containerStyle}
-        data-testid="container"
-        id={htmlId}
+  if (imageUrl) {
```

Comment:
Invert this condition.

Read more about this:
https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

Leave it like:

```js
if (!imageUrl) {
  return null
}

return (
  <LinkWrapper
```